pickATrip_django_apps/templates/pickATrip_django_apps/base.html

```diff
@@ -21,20 +21,41 @@
 			</div>
 		{% endfor %}
 	{% endif %}
+<<<<<<< HEAD
 	
 	<div id ="content" class = "container">
```

Comment:
that's the update clock content you need to decide where you want it on the .HTML page bu you can't have two with the same name, the site doesn't go up for me for that reason

Reply:
Where do I have 2 with the same name?